packages/website/static/css/custom.css

```diff
@@ -28,3 +28,11 @@
 a, a:hover {
   text-decoration: underline;
 }
+
+.imageAlignTop .blockImage {
+  max-width: 300px;
+}
+
+code {
+  text-align: left;
+}
```

Comment:
```suggestion
}

.gridBlock {
  align-items: flex-end;
}

.twoByGridBlock {
  margin-bottom: 36px;
}
```

This helps align the content of the grid blocks. But not a blocker for shipping. 🚀
<img width="1517" alt="screen shot 2018-12-05 at 11 01 58 am" src="https://user-images.githubusercontent.com/3051193/49526151-38106880-f87d-11e8-850d-5b33af994d9e.png">